### ISSUE TITLE
✨ Markdown Editor in Storybook - for job poster

### ISSUE THREAD

**Issue by gobyrne** [GCTC-NTGC/gc-digital-talent#7766]
## ✨ Feature

On the job poster users are inputting lists and links, but they are not marked up properly. Instead the are just plain text which is a textbook accessibility fail.

## 🕵️ Details

This issue is to implement a markdown editor in storybook. Once we have it there we will pass to design to decide how we want to present the component.

## 🙋‍♀️ Proposed Implementation

Eric has used [TipTap](https://tiptap.dev/) in the past to good effect. (it's light weight and let's you add things in as you need them)

## 🌎 Localization

(optional) Provide any new copy along with translations available.

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the feature was properly implemented.

- [ ] Markdown editor working in Storybook with a save button that then presents the text back as it would appear (or similar)
- [ ] Markdown lists and links only
- [ ] TipTap is a suggestion but not the only option
